backend/account/constants.py

```diff
@@ -67,8 +67,15 @@ class PluginConfig:
 
 
 class AuthorizationErrorCode:
-    """Error code reference
-    frontend/src/components/error/GenericError/GenericError.jsx."""
+    """Error codes
```

Comment:
@muhammad-ali-e shouldn't these be part of the plugin ideally?
CC: @hari-kuriakose

Reply:
This PR doesn't have much changes here except here added the docstring.
I feels like both ways,
 Is this part of a plugin? It seems to be utilized exclusively for multi-tenancy (currently enterprise-only).
 Is this part of OSS? It incorporates certain UI components and could potentially be reused with other plugins and OSS.
 @johnyrahul